postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -385,7 +400,7 @@ def _run_explain_safe(self, dbname, statement, obfuscated_statement):
             )
             return None, DBExplainError.database_error, e
 
-    def _collect_plan_for_statement(self, row):
+    def _collect_plan_for_statement(self, max_query_size, row):
         try:
             obfuscated_statement = datadog_agent.obfuscate_sql(row['query'])
```

Comment:
Is there a decent chance of truncation causing queries to fail to obfuscate? If so maybe we need to send some signal in this case as well.

Reply:
We covered this during the daily but just to capture what we said: yes there's a chance that truncation would fail the obfuscation but those cases are very limited (one example would be if a `!=` was truncated and only the `!` was left..this would be considered an invalid negative equal operator).